lib/act-fluent-logger-rails/logger.rb

```diff
@@ -74,7 +74,12 @@ def add(severity, message = nil, progname = nil, &block)
 
     def add_message(severity, message)
       @severity = severity if @severity < severity
-      @messages << message
+      @messages << encode_message(message)
```

Comment:
Thanks for reviewing!
No problem with this code I think. I fixed the code and pushed it.